### PR TITLE
* added exercise-05; maintenance

### PR DESCRIPTION
**Pull request by HolQue**

### CODE REVIEW

File: 900_building_testsuites/doc_src/building_testsuites.rst [hunk 1]
```diff
@@ -295,7 +295,7 @@ access the configuration file in a certain order: Level 1 has the highest priori
 * Level 2 (**recommended**)
 
   The name of the variant is provided in command line of the RobotFramework AIO. This requires an additional configuration file
-  that contains the mapping between the variant and the variant specific configuration
+  that contains the mapping between the variant name and the variant specific configuration
   (like described in `How does the content of configuration files in JSON format look like?`_).
 
   This also requires that this mapping file is known to the test (path and name is an input parameter of the ``Suite Setup``).
```
Comment (test-fullautomation) [inline 900_building_testsuites/doc_src/building_testsuites.rst]: Hi Holger,
please add something similar:
"The reason is that using level 2 you explicitly mention in each robot file which configuration file(s) need to be used. This is clearly understandable, traceable and scales for huge test suites."
Thank you,
 Thomas

File: 900_building_testsuites/exercise-05/exercise-05.robot [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 *** Settings ***
 
+# coming soon # Library    RobotFramework_TestsuitesManagement    WITH NAME    testsuites
```
Comment (test-fullautomation) [inline 900_building_testsuites/exercise-05/exercise-05.robot]: I wonder if we should write e.g.
```
RobotFramework_TestsuitesManagement WITH NAME tm
```
??
```
tm.testsuite_setup ../config/exercise-04_variants.json
```
is vey smart...

**Review by test-fullautomation — APPROVED** [review]
Hi Holger,
looks good to me!
Thank you,
Thomas

File: 900_building_testsuites/doc_src/building_testsuites.rst (hunk 1 above)
Comment (test-fullautomation) [reply 900_building_testsuites/doc_src/building_testsuites.rst]: ping...

File: 900_building_testsuites/exercise-05/exercise-05.robot (hunk 2 above)
Comment (test-fullautomation) [reply 900_building_testsuites/exercise-05/exercise-05.robot]: ping...

File: 900_building_testsuites/doc_src/building_testsuites.rst (hunk 1 above)
Comment (HolQue) [reply 900_building_testsuites/doc_src/building_testsuites.rst]: Hi Thomas,

this proposal is confusing. In every level 1-3 something is mentioned explicitly. In every case the outcome is logged - to support understandability and traceability. In level 2 the robot file only contains a link to a variants configuration file. Only together with the name of a variant in command line we have a complete information. All these things are already explained in detail in this tutorial. I do not understand why we need such an additional sentence. At least this sentence is not a plausible reason for especially using level 2.

File: 900_building_testsuites/exercise-05/exercise-05.robot (hunk 2 above)
Comment (HolQue) [reply 900_building_testsuites/exercise-05/exercise-05.robot]: Hi Thomas,

rather short abbreviation. But also OK. I will change.

File: 900_building_testsuites/doc_src/building_testsuites.rst (hunk 1 above)
Comment (test-fullautomation) [reply 900_building_testsuites/doc_src/building_testsuites.rst]: I resolve this issue based on the result of our talk.
New documentation is O.K. now.

File: 900_building_testsuites/exercise-05/exercise-05.robot (hunk 2 above)
Comment (test-fullautomation) [reply 900_building_testsuites/exercise-05/exercise-05.robot]: nice! I also like that at least I associate nothing with an object called tm.

**Pull request merged**